securedrop/tests/test_alembic.py

```diff
@@ -104,9 +102,7 @@ def test_alembic_head_matches_db_models(journalist_app,
     '''
     models_schema = get_schema(journalist_app)
 
-    config.DATABASE_FILE = config.DATABASE_FILE + '.new'
```

Comment:
Had to ask about this change - apparently the updated version of pytest required only allows fixtures as decorators, hence the simplification.